README.md

```diff
@@ -0,0 +1,19 @@
+# Intel Retail Documentation Repository
+
+![Requirements Review Process Flow](./requirements-review-process-flow.png)
+
+## Requirements Process Breakdown
+
+- Submit a github issue and follow the requirements template.
+
+- Once a week there will be a requirements review meeting where we will approve, reject, or ask for modification to the requirement.
+
+- If approved the requirement will be moved into the design stage in the documents project.
+
+- A detailed design document will be made by the submitter and development team
```

Comment:
can we share any design document template to follow ? may be edgex one ?

Reply:
The template was roughly based on the EdgeX template but we can add/remove what makes sense.